File: Problems/Furthest-Point-From-Origin/Furthest-Point-From-Origin.java
class Solution {
    public int furthestDistanceFromOrigin(String moves) {
        int[] ct = {0, 0, 0};
        for (char c : moves.toCharArray())
            switch (c) {
                case 'L': ct[1]++; break;
                case 'R': ct[2]++; break;
                case '_': ct[0]++;
            }
        return ct[0] + Math.abs(ct[1] - ct[2]);
    }
}

File: Problems/Furthest-Point-From-Origin/Furthest-Point-From-Origin.js
/**
 * @param {string} moves
 * @return {number}
 */
var furthestDistanceFromOrigin = function(moves) {
    let ct = [0, 0, 0];
    for (let i = 0; i < moves.length; i++)
        switch (moves.charAt(i)) {
            case 'L': ct[1]++; break;
            case 'R': ct[2]++; break;
            case '_': ct[0]++;
        }
    return ct[0] + Math.abs(ct[1] - ct[2]);
};
